ocaml/idl/datamodel.ml

```diff
@@ -174,8 +174,7 @@ let _vdi_nbd_server_info = "vdi_nbd_server_info"
 let _pusb = "PUSB"
 let _usb_group = "USB_group"
 let _vusb = "VUSB"
-
-
+let _net_sriov = "NET_sriov"
```

Comment:
I believe `schema_minor_vsn` should be incremented.

Reply:
indeed